### ISSUE TITLE
Is there a straight way to get metadata of a Hive table

### ISSUE THREAD

**Issue by tinylambda** [dropbox/PyHive#263]
I means not only the column - type - comment information, but also the other attributes of a Hive table, like LOCATION, OUTPUTFORMAT, TBLPROPERTIES, PARTITIONED BY and so on.

Thanks a lot.

**Comment by AntoniosHadji** [comment]
`show create table tablename`  shows the columns and the other attributes.

**Comment by tinylambda** [comment]
> `show create table tablename` shows the columns and the other attributes.

yes, it's a straight way to get the information by using this method, but the result is not structured and need some effort to parse it and extract related keys and values.
